### PR TITLE
showed incomplete orders

### LINKED ISSUE #3: REPORT: Incomplete orders
As a business analyst, I would like to see an HTML report showing all orders that have not been paid for yet.

* Order Id
* Customer name
* Total cost of all items on the order

### PR DESCRIPTION
**Pull request by andrew-webb07**
Description of PR that completes issue here...

## Changes

- nullorders.py view to show unpaid orders
- list_of_null_orders

## Requests / Responses

If this PR contains code that defines a new request/response, or changes an existing one, please put the JSON representations here.

**Request**

## Testing

Description of how to test code...

- [ ] Run migrations
- [ ] Run test suite
- [ ] Seed database
-git fetch all
-git ticket-3-incomplete-orders
- Put http://localhost:8000/reports/nullorders into your browser

## Related Issues

- Fixes #3

### CODE REVIEW

File: bangazonreports/views/users/nullorders.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+                auth_user.first_name || " " || auth_user.last_name AS customer_name
+                FROM bangazonapi_Order
+                JOIN bangazonapi_OrderProduct
+                ON bangazonapi_OrderProduct.order_id = bangazonapi_order.id
+                JOIN bangazonapi_Product
+                ON bangazonapi_Product.id = bangazonapi_OrderProduct.product_id
+                JOIN auth_user
+                ON auth_user.id = bangazonapi_Order.customer_id
+                WHERE bangazonapi_order.payment_type_id IS NULL
+                GROUP BY auth_user.first_name
```
Comment (hannahhall) [inline bangazonreports/views/users/nullorders.py]: What if there's two users with the same first name in the database?

**Review by hannahhall — CHANGES_REQUESTED** [review]
Looks good, just need to update the sql query

File: bangazonreports/views/users/nullorders.py (hunk 1 above)
Comment (andrew-webb07) [reply bangazonreports/views/users/nullorders.py]: GROUP BY bangazonapi_order.id?

Comment (hannahhall) [reply bangazonreports/views/users/nullorders.py]: Yeah that will work

**Review by hannahhall — APPROVED** [review]

**Pull request merged**